atividade03/src/main/java/br/edu/uepb/eda/atividade03/PilhaComLista.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
+ * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
+ */
+package br.edu.uepb.eda.atividade03;
+
+/**
+ *
+ * @author jeffgabriiel
+ */
+
+public class PilhaComLista implements Pilha_IF {
+	
+	private ListaEncadeada list;
+	private int max_element = 10;
```

Comment:
Boa! é uma boa prática de programação atribuir significado a valores constantes que poderia torna o código mais inelegivel, apenas um sugestão, no caso de valores constantes poderia ser escrito todo em maiuscula separado por underline
```suggestion
	private final int MAX_ELEMENT = 10;
```